cmd/deploy.go

```diff
@@ -566,6 +566,20 @@ func (c deployConfig) Prompt() (deployConfig, error) {
 	}
 
 	var qs = []*survey.Question{
+		{
+			Name: "remote",
+			Prompt: &survey.Input{
+				Message: "Build on remote cluster",
+				Default: strconv.FormatBool(c.Remote),
```

Comment:
I think this will require that the user literally type "yes", "true" etc.
Perhaps take a look at the documentation for the library we are using for this "https://github.com/AlecAivazis/survey", or other commands which prompt for boolean values.  A good choice for boolean values is  `survey.Confirm` for 
```
Name: "remote",
Prompt: &survey.Confirm{
    Message: "Trigger a remote (on-cluster) build?",
    Default: c.Remote,
}
```